kvclientmanager.go

```diff
@@ -285,6 +286,22 @@ func (m *kvClientManager) Close() error {
 	return nil
 }
 
+type KvClientError struct {
```

Comment:
Aren't our errors typically in the errors file?

Reply:
Not always.  Admittedly, it's actually a practice I'm not entirely fond of, but admittedly I also don't like having them scattered _everywhere_ either... Hmm.